package.json

```diff
@@ -6,14 +6,16 @@
     "build": "npx babel src --out-dir lib",
     "test": "nyc --reporter=text --reporter=html mocha --recursive --require @babel/polyfill --require @babel/register src/test",
     "coverage": "nyc  report --reporter=text-lcov --reporter=lcov",
-    "start": "node lib/index.js",
+    "start": "npx babel-node lib/index.js",
```

Comment:
and you need to add a migration script in start script

Reply:
I finished them, you review them now